lstchain/io/io.py

```diff
@@ -1018,7 +1025,29 @@ def read_dl2_to_pyirf(filename):
     events = pd.read_hdf(filename, key=dl2_params_lstcam_key).rename(columns=name_mapping)
     events = table.QTable.from_pandas(events)
 
+    #Make the columns as Quantity
     for k, v in unit_mapping.items():
         events[k] *= v
 
-    return events, pyirf_simu_info
\ No newline at end of file
+    return events, pyirf_simu_info
+
+def read_data_dl2_to_QTable(filename):
+    """
+    Read data DL2 files from lstchain and return QTable format
+
+    Parameters
+    ----------
+    filename: path to the lstchain DL2 file
+    Returns
+    -------
+    `astropy.table.QTable`
+    """
+    #Mapping
+    name_mapping = {
+        "gammaness" : "gh_score"
+    }
+    #TODO: Update other parameters with keys
+    data = pd.read_hdf(filename, key = dl2_params_lstcam_key).rename(columns=name_mapping)
+    data = table.QTable.from_pandas(data)
```

Comment:
units?

Reply:
I will add them now. Was earlier adding the units in the main scripts, but will update them.